### ISSUE TITLE
iOS app fails to post image when server is running on Bluemix

### ISSUE THREAD

**Issue by kurtismgit** [IBM-Swift/BluePic#306]
Yesterday I attempted to setup BluePic using the manual path.  When I eventually got to the end, the part that isn't working is posting a new image to to the server.  Looks like the request that does POST /image fails for some reason.  I'm attaching the logs from both the client and server at the time of the failure.  The client is launched on a simulator from within XCode.

It's also worth noting that I'm able to run the server locally and the POST /image request works.

Please let me know what additional information is needed to help debug the problem.

**Comment by kurtismgit** [comment]
[bluepic_client.txt](https://github.com/IBM-Swift/BluePic/files/413361/bluepic_client.txt)
[bluepic_server.txt](https://github.com/IBM-Swift/BluePic/files/413360/bluepic_server.txt)
[bluepic_staging.txt](https://github.com/IBM-Swift/BluePic/files/413362/bluepic_staging.txt)

**Comment by tfrank64** [comment]
Authentication seems to work in iOS app, but posting an image fails for me with the following error: `ERROR: parse IncomingHTTPSocketProcessor.swift line 117 - Failed to parse a request`

**Comment by tfrank64** [comment]
Remember to re-enable the deployment tracker when https://github.com/IBM-Swift/Kitura/issues/686 is resolved.

**Comment by kurtismgit** [comment]
@tfrank64 I pulled master and redeployed.  I no longer see an error when posting an image.  Feel free to resolve this issue.  Unfortunately now the whisk /weather action fails.  I'll file a separate issue for that.

**Comment by tfrank64** [comment]
I'm still having that failure to parse issue so I will leave this open as we wait for more info on Kitura issue.

**Comment by tfrank64** [comment]
When I have time, I will attempt to use a later version of Kitura and see if the issue still persists. The issue seems to happen with 0.24 versions.

**Comment by tfrank64** [comment]
After more troubleshooting, I'm told newer versions of Kitura might fix this issue. I'm currently waiting on other BluePic dependencies to be updated to 09/07.

**Comment by tfrank64** [comment]
Seems like posting an image still doesn't work with Kitura 0.32.
More specifically, its the creating/getting of the user that fails.

**Comment by tfrank64** [comment]
There seems to be a fix now that is currently being tested in Kitura-net, will integrate once it's tagged.

**Comment by tfrank64** [comment]
It's been tagged, still works great. Closing issue
